### PR TITLE
Migrate to v0.14 API

### PR DESCRIPTION
**Pull request by cosmo0920**

### CODE REVIEW

**Comment by okkez** [comment]
You can use `timer_execute` plugin helpr in `in_sql`.

May need to replace `#emit` with `#process`?
https://github.com/cosmo0920/fluent-plugin-sql/blob/7e6f29f88037309fc961f9d95dfaaa236af6bb59/lib/fluent/plugin/out_sql.rb#L210

File: lib/fluent/plugin/out_sql.rb [hunk 1]
```diff
@@ -119,15 +112,15 @@ def one_by_one_import(records)
           begin
             @model.import([record])
           rescue ActiveRecord::StatementInvalid, ActiveRecord::Import::MissingColumnError => e
-            @log.error "Got deterministic error again. Dump a record", :error => e.message, :error_class => e.class, :record => record
+            @log.error "Got deterministic error again. Dump a record", error: e.message, error_class: e.class, record: record
```
Comment (tagomoris) [inline lib/fluent/plugin/out_sql.rb]: On v0.14, logger shows error class and its message just by `error: e`.

**Comment by cosmo0920** [comment]
> May need to replace #emit with #process?
https://github.com/cosmo0920/fluent-plugin-sql/blob/7e6f29f88037309fc961f9d95dfaaa236af6bb59/lib/fluent/plugin/out_sql.rb#L210

@okkez This plugin is synchronous buffered plugin: https://github.com/cosmo0920/fluent-plugin-sql/blob/7e6f29f88037309fc961f9d95dfaaa236af6bb59/lib/fluent/plugin/out_sql.rb#L7-L9

**Comment by cosmo0920** [comment]
Any progress?

**Comment by repeatedly** [comment]
Sorry for the late. Thanks!

**Comment by cosmo0920** [comment]
Thank you! :yum:

**Pull request merged**